static/compiler.js

```diff
@@ -808,10 +814,16 @@ Compiler.prototype.updateButtons = function () {
     // Disable any of the options which don't make sense in binary mode.
     var noBinaryFiltersDisabled = !!filters.binary && !this.compiler.supportsFiltersInBinary;
     this.noBinaryFiltersButtons.prop('disabled', noBinaryFiltersDisabled);
+
+    this.filterLabelsButton.prop('disabled', this.compiler.disabledFilters.includes('labels'));
```

Comment:
As state above, I do think we have a way to already do this. 
Also note that `.includes`, while widely supported, might be a bad choice here. Prefer using `.indexOf > 0`

Reply:
Did I warn I know next to nothing about JS? :-)
Thanks a lot for the tips!